controls/srg_gpos/SRG-OS-000420-GPOS-00186.yml

```diff
@@ -8,4 +8,5 @@ controls:
         rules:
             - sysctl_net_ipv4_tcp_invalid_ratelimit
             - sysctl_net_ipv4_tcp_syncookies
+            - sysctl_net_ipv4_tcp_invalid_ratelimit_value=five_hundred
```

Comment:
So in the current form of the script the value needs to be the value you want on the spreadsheet. Below is the current text.

```The command should output the following line:
/etc/sysctl.conf:net.ipv4.tcp_invalid_ratelimit = five_hundred```

Reply:
I'm sorry, I don't understand. Can you rephrase? What is the expected change here?

Reply:
So the issue is #8612 so we can merge it as but, we have to fix the export script.

Reply:
Aha, yes, now I understand. I run into the same problem 2 weeks ago in the PR https://github.com/ComplianceAsCode/content/pull/8555.

Reply:
#8612 has been fixed, so this should work once the branch is rebased.